python/lsst/ts/dream/csc/mock/mock_dream.py

```diff
@@ -152,23 +156,23 @@ async def exit(self) -> None:
         await self.close()
         self.log.info("Done closing")
 
-    async def resume(self):
+    async def resume(self) -> None:
         """Indicate that DREAM is permitted to resume automated operations."""
         self.log.info("resume called.")
 
-    async def open_hatch(self):
+    async def open_hatch(self) -> None:
         """Open the hatch if DREAM has evaluated that it is safe to do so."""
         self.log.info("open called.")
 
-    async def close_hatch(self):
+    async def close_hatch(self) -> None:
         """Close the hatch."""
         self.log.info("close called.")
 
-    async def stop(self):
+    async def stop(self) -> None:
         """Immediately stop operations and close the hatch."""
         self.log.info("stop called.")
 
-    async def ready_for_data(self, ready: bool):
+    async def ready_for_data(self, ready: bool) -> None:
```

Comment:
This got renamed no?

Reply:
Yes, I was waiting for you to finish your review of `ts_dream_common`. Done now.